lib/puma/server.rb

```diff
@@ -428,8 +429,8 @@ def handle_servers
         STDERR.puts "Exception handling servers: #{e.message} (#{e.class})"
         STDERR.puts e.backtrace
       ensure
-        @check.close
-        @notify.close
+        @check.close  rescue nil
```

Comment:
What was the motivation for adding rescue nil?

Reply:
Exactly, I don't recall.  We're in an ensure clause at shutdown ?